packages/snitches/package.json

```diff
@@ -24,7 +24,8 @@
   },
   "files": [
     "dist",
-    "README.md"
+    "README.md",
+    "LICENSE"
```

Comment:
I don't think this is right, these paths are relative to the package.json file - `LICENSE` is at the top level of the project. I think we either need a second LICENSE file here in this directory, or this needs to be `"../../LICENSE"`